redpanda/README.md

```diff
@@ -0,0 +1,134 @@
+# Agent Check: Redpanda
+
+## Overview
+
+This Datadog-[Redpanda][1] integration collects key metrics from Redpanda by default. It can also be configured to add additional metric groups based on specific user needs.
+
+Redpanda is a Kafka API compatible streaming platform for mission-critical workloads.
+
+## Setup
+
+### Installation
+
+First, [download and launch the Datadog Agent][9].
+
+Then, manually install the Redpanda check. [Instructions vary depending on the environment][10].
+
+> Current Redpanda integration version: `1.0.0`
+
+#### Host
+
+Run `datadog-agent integration install -t datadog-redpanda==<INTEGRATION_VERSION>`.
+
+#### Containerized
+
+The best way to use this integration with the Docker Agent is to build the Agent with this integration installed. Use the following Dockerfile to build an updated version of the Agent:
+
+```dockerfile
+FROM gcr.io/datadoghq/agent:latest
+
+ARG INTEGRATION_VERSION=1.0.0
+
+RUN agent integration install -r -t datadog-redpanda==${INTEGRATION_VERSION}
+```
+
+Build the image and push it to your private Docker registry.
+
+Then, upgrade the Datadog Agent container image. If the Helm chart is used, modify the `agents.image` section in the `values.yaml` to replace the default agent image:
+
+```yaml
+agents:
+  enabled: true
+  image:
+    tag: <NEW_TAG>
+    repository: <YOUR_PRIVATE_REPOSITORY>/<AGENT_NAME>
+```
+
+Use the new `values.yaml` to upgrade the Agent:
+
+```shell
+helm upgrade -f values.yaml <RELEASE_NAME> datadog/datadog
+```
+
+### Configuration
+
+#### Host
```

Comment:
I recommend taking a look at the **Configuration** section in [TCP Check](https://docs.datadoghq.com/integrations/tcp_check/?tab=containerized) and using the following outline to distinguish between the Host and Containerized sections.

```suggestion
### Configuration

<!-- xxx tabs xxx -->
<!-- xxx tab "Host" xxx -->

#### Host

<!-- xxz tab xxx -->
<!-- xxx tab "Containerized" xxx -->

#### Containerized

<!-- xxz tab xxx -->
<!-- xxz tabs xxx -->
```

Reply:
I've added tabs for the installation and configuration. Thanks for the suggestion!